charts/opentelemetry-operator/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: opentelemetry-operator
-version: 0.24.0
+version: 0.25.0
```

Comment:
This needs to be 0.24.1 (and run `make generate-examples` again) . We should also update the [docs](https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-operator/CONTRIBUTING.md) to specify patch version and not minor.